### DISCUSSION TITLE
Collaboration between Mobilizon and pretalx?

### DISCUSSION THREAD

**Discussion by linux-lukas** [pretalx/pretalx#1195] [Ideas] [answered]
I was intensively involved with Fediverse for the purpose of an honorary office, because in the honorary office the events are unfortunately still organised via Facebook event entries with Mobilizon as a possible alternative. 

This gave me the idea of whether interoperability between pretalx and Mobilizon might be useful?

**Comment by rixx** [comment, marked as answer]
I'm not sure how such an interoperability would look like on our end, but as always – this sounds like a great case for a plugin, for example an exporter plugin.

**Reply by linux-lukas** [reply]
I could imagine that one could deposit a Mobilizon instance in pretalx (optional) and a Mobilizon event could be created for the event on the deposited instance incl. corresponding access data for an account on this instance. @rixx

**Reply by rixx** [reply]
Sure, that sounds like a great case for a plugin if anybody wants to work on that! Plugin development is documented [here](https://docs.pretalx.org/developer/plugins/index.html).
